src/Computers.java

```diff
@@ -0,0 +1,20 @@
+public class Computers {
+    private String name;
+    private String brand;
+    private String type;
+    private int screenSize;
+    private int numberOfCores;
+
+    public Computers(String name, String brand, String type, int screenSize, int numberOfCores) {
+        this.name = name;
+        this.brand = brand;
+        this.type = type;
+        this.screenSize = screenSize;
+        this.numberOfCores = numberOfCores;
+    }
```

Comment:
According to task, you should create constructor with only 2 mandatory properties `String brand`, `String screenSize`. The reason of that is ability to create instance of Computers class with only two parameters. Other parameters you can additionaly set up with appropriate "Setters" methods. For example 
```
public Computers setName(String name) {
        this.name = name;
        return this;
    }
```